package/piwebcam/camera.txt

```diff
@@ -3,5 +3,9 @@
 # Use v4l2-ctl -L or have a look at camera.txt.example
 # to view all possible settings
 
-auto_exposure_bias=3
+auto_exposure=0
+auto_exposure_bias=12
```

Comment:
I think we could use 13 or 14 here as well? What do you think?

Reply:
This is "good enough" for me.